### PR TITLE
Replaced utils.timeout with timeout_decorator

### PR DESCRIPTION
**Pull request by k2fong**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/183)

### CODE REVIEW

File: mobly/controllers/monsoon.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3.4
```
Comment (xpconanfan) [inline mobly/controllers/monsoon.py]: why adding this?

Comment (k2fong) [inline mobly/controllers/monsoon.py]: hmm.  I didn't add this.  But seems like most of the other files have it this too.  I wonder if it was automatically added somehow.

Comment (xpconanfan) [inline mobly/controllers/monsoon.py]: It seems like you did not start with the latest code as base.

Comment (k2fong) [inline mobly/controllers/monsoon.py]: it should be sync'd, I pulled it right before I started this.  They seems to be both at commit 107 (Apr 7, 2017)

Comment (xpconanfan) [inline mobly/controllers/monsoon.py]: If you are on the right base and you didn't add this, then this should be here.
Pls remove.

**Comment by k2fong** [comment]
Added updates to the setup.py with timeout_decorator as install dependency.

File: mobly/controllers/monsoon.py (hunk 1 above)
Comment (k2fong) [inline mobly/controllers/monsoon.py]: removed

File: mobly/utils.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3.4
```
Comment (xpconanfan) [inline mobly/utils.py]: remove

Comment (k2fong) [inline mobly/utils.py]: done

**Comment by k2fong** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[mobly/utils.py, line 1 at r3](https://reviewable.io:443/reviews/google/mobly/183#-KhinC_Gaf3dSSBcPmg1-r3-1:-KhixYxr_-Zvz2xsplKV:b-896fix) ([raw file](https://github.com/google/mobly/blob/e0973aaa50c4fea620142444b918db238219243d/mobly/utils.py#L1)):*
<details><summary><i>Previously, k2fong wrote…</i></summary><blockquote>

done
</blockquote></details>

Done.

---

*[mobly/controllers/monsoon.py, line 1 at r1](https://reviewable.io:443/reviews/google/mobly/183#-KhinC_Gaf3dSSBcPmg0-r1-1:-Khix_G_hEHUfCVOnsof:b-896fix) ([raw file](https://github.com/google/mobly/blob/29d2c783d37e028871e7511bb286aa6bc00629cb/mobly/controllers/monsoon.py#L1)):*
<details><summary><i>Previously, k2fong wrote…</i></summary><blockquote>

removed
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly/183)*

**Comment by adorokhine** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2, 1 of 1 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly/183#-:-Khj29SHo-Bv-1OshVAO:bnfp4nl)*

**Pull request merged**